internal/backend/db/dbgorm/scheme/scheme.go

```diff
@@ -57,15 +51,15 @@ var Tables = []any{
 }
 
 type Build struct {
-	BuildID   string `gorm:"primaryKey"`
+	BuildID   sdktypes.UUID `gorm:"primaryKey;type:uuid"`
```

Comment:
why `sdktypes.UUID` and not google's UUID?

Reply:
IDK, you exposed it, I assumed this is what we want to use